Tools-Override/tests.targets

```diff
@@ -55,11 +60,72 @@
 
     <TestCommandLine Condition="'$(Performance)'!='true'">$(TestProgram) $(TestArguments) {XunitTraitOptions}</TestCommandLine>
   </PropertyGroup>
+  
+  <!-- This will generate a version of the dotnet cli that runs with the live
+       build of corefx using the other components gathered from the stable 
+       version consumed from the Tools directory 
+  -->
+  <Target Name="ConstructDotNetExe" Condition="!Exists('$(BinDir)dotnetcli')" >
```

Comment:
The current approach was to not add an additional probing path to dotnet.exe but instead just overwrite the dependencies in the shared framework that we built locally. So we copy whatever we have existing in the Tools dir and overwrite the deps.